### PR TITLE
Check configured ports per bond

### PR DESCRIPTION
**Pull request by cletomartin**
This will check whether switch to which node are plugged is correctly
configured.

BugzId: 61000

### CODE REVIEW

File: tests/network.bash [hunk 1]
```diff
@@ -15,10 +16,22 @@ preflight_test () {
...
     fi
 
+    if [ -z $4 ]; then
+        if [ $3 -eq 0 ]; then
+            portsPerBond=0
+        else
+            portsPerBond=$(($nics / 2))
+        fi
+    else
+        portsPerBond=$3
```
Comment (mattwhite) [inline tests/network.bash]: `portsPerBond=$4`

File: tests/network.bash [hunk 2]
```diff
@@ -15,10 +16,22 @@ preflight_test () {
         bonds=$3
     fi
 
+    if [ -z $4 ]; then
+        if [ $3 -eq 0 ]; then
+            portsPerBond=0
+        else
+            portsPerBond=$(($nics / 2))
```
Comment (mattwhite) [inline tests/network.bash]: `portsPerBond=$(($nics / $bonds))`

File: tests/network.bash [hunk 3]
```diff
@@ -33,8 +46,24 @@ preflight_test () {
...
+
+        if [ ! -d /proc/net/bonding ]; then
+            output_fail "Unable to find /proc/net/bonding directory"
+            fail=1
+        elif [ $(ls /proc/net/bonding/* | wc -l) -ne $bonds ]; then
+            output_fail "Missing bond interface descriptors at /proc/net/bonding"
+            fail=1
         else
-            output_ok "$status"
+            for f in $(ls /proc/net/bonding/*); do
```
Comment (mattwhite) [inline tests/network.bash]: `for f in /proc/net/bonding/*; do` works too.

**Comment by mattwhite** [comment]
Looks good after the above comments.

**Comment by cletomartin** [comment]
Thanks Matt! Updated with your comments.

**Pull request merged**